### PR TITLE
Change RoBERTa and XLM-RoBERTa's Format to Mirror BERT's

### PR DESCRIPTION
**Pull request by abheesht17**
Resolves https://github.com/keras-team/keras-nlp/issues/403
Resolves https://github.com/keras-team/keras-nlp/issues/405

Children:
https://github.com/abheesht17/keras-nlp/pull/1 (Resolves https://github.com/keras-team/keras-nlp/issues/414)

### CODE REVIEW

File: keras_nlp/models/roberta/roberta_models.py [hunk 1]
```diff
@@ -26,17 +26,18 @@ def roberta_kernel_initializer(stddev=0.02):
 
 
 @keras.utils.register_keras_serializable(package="keras_nlp")
-class RobertaCustom(keras.Model):
-    """RoBERTa encoder with a customizable set of hyperparameters.
+class Roberta(keras.Model):
+    """RoBERTa encoder with custom hyperparameters.
```
Comment (jbischof) [inline keras_nlp/models/roberta/roberta_models.py:30]: Drop "custom" language

File: keras_nlp/models/roberta/roberta_tasks.py [hunk 2]
```diff
@@ -55,19 +52,13 @@ class RobertaClassifier(keras.Model):
...
 
     def __init__(
-        self,
-        backbone,
-        num_classes,
-        hidden_dim=None,
-        dropout=0.0,
-        name=None,
-        trainable=True,
+        self, backbone, num_classes, hidden_dim=None, dropout=0.0, **kwargs
```
Comment (jbischof) [inline keras_nlp/models/roberta/roberta_tasks.py:55]: Not something you changed, but is `0` the correct default for dropout? If so, why are we including it?

File: keras_nlp/models/roberta/roberta_models.py [hunk 3]
```diff
@@ -86,12 +85,10 @@ def __init__(
         intermediate_dim,
         dropout=0.1,
         max_sequence_length=512,
-        name=None,
-        trainable=True,
+        **kwargs,
     ):
+        start_token_index = 0
```
Comment (jbischof) [inline keras_nlp/models/roberta/roberta_models.py:90]: You could remove this line and define this constant directly in line 154 since it's not used in the encoder directly.

File: keras_nlp/models/xlm_roberta/xlm_roberta_models.py [hunk 4]
```diff
@@ -21,19 +21,19 @@
 
 
 @keras.utils.register_keras_serializable(package="keras_nlp")
-class XLMRobertaCustom(roberta_models.RobertaCustom):
-    """XLM-RoBERTa encoder with a customizable set of hyperparameters.
+class XLMRoberta(roberta_models.Roberta):
+    """XLM-RoBERTa encoder with custom hyperparameters.
```
Comment (jbischof) [inline keras_nlp/models/xlm_roberta/xlm_roberta_models.py:25]: Drop "custom" language.

File: keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py [hunk 5]
```diff
@@ -22,83 +22,82 @@
...
+    single input tensor. If a single tensor is passed, it will be packed
+    equivalently to a tuple with a single element.
+
+    The SentencePiece tokenizer can be accessed via the `tokenizer` property on
+    this layer, and can be used directly for custom packing on inputs.
+
+    Args:
+        proto: Either a `string` path to a SentencePiece proto file, a `bytes`
+            object with a serialized SentencePiece proto.
+        sequence_length: The length of the packed inputs. Only used if
```
Comment (jbischof) [inline keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py:57]: Remove notes about `pack_inputs`, and arg we removed a while ago.

File: keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py [hunk 6]
```diff
@@ -22,83 +22,82 @@
...
+            `pack_inputs` is True.
+        truncate: The algorithm to truncate a list of batched segments to fit
```
Comment (jbischof) [inline keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py:59]: Remove `pack_inputs` note here too

File: keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py [hunk 7]
```diff
@@ -22,83 +22,82 @@
...
+
+    This preprocessing layer will do three things:
+
+    - Tokenize any number of inputs using a
+    `keras_nlp.tokenizers.SentencePieceTokenizer`.
+    - Pack the inputs together with the appropriate `"<s>"`, `"</s>"` and
+    `"<pad>"` tokens, i.e., adding a single `"<s>"` at the start of the
+    entire sequence, `"</s></s>"` at the end of each segment, save the last and
+    a `"</s>"` at the end of the entire sequence.
+    - Construct a dictionary of with keys `"token_ids"`, `"padding_mask"`, that
```
Comment (jbischof) [inline keras_nlp/models/xlm_roberta/xlm_roberta_preprocessing.py:38]: the keys "token_ids" and "padding_mask" that

**Review by jbischof — APPROVED** [review]
Great work @abheesht17! Just some small comments to address then ship away.

File: keras_nlp/models/roberta/roberta_tasks.py (hunk 2 above)
Comment (abheesht17) [reply keras_nlp/models/roberta/roberta_tasks.py:55]: Yep.

https://github.com/facebookresearch/fairseq/blob/b5a039c292facba9c73f59ff34621ec131d82341/fairseq/models/roberta/model.py#L637

https://github.com/facebookresearch/fairseq/blob/b5a039c292facba9c73f59ff34621ec131d82341/fairseq/models/roberta/model.py#L515

Comment (abheesht17) [reply keras_nlp/models/roberta/roberta_tasks.py:55]: HF, on the other hand, has 0.1:

https://huggingface.co/roberta-base/blob/main/config.json#L9

https://github.com/huggingface/transformers/blob/bd469c40659ce76c81f69c7726759d249b4aef49/src/transformers/models/roberta/modeling_roberta.py#L1437-L1439

File: keras_nlp/models/roberta/roberta_models.py [hunk 8]
```diff
@@ -26,17 +26,18 @@ def roberta_kernel_initializer(stddev=0.02):
 
 
 @keras.utils.register_keras_serializable(package="keras_nlp")
-class RobertaCustom(keras.Model):
-    """RoBERTa encoder with a customizable set of hyperparameters.
+class Roberta(keras.Model):
+    """RoBERTa encoder with hyperparameters.
```
Comment (jbischof) [inline keras_nlp/models/roberta/roberta_models.py:30]: Sorry this should just say `"""RoBERTa encoder"""`.

File: keras_nlp/models/xlm_roberta/xlm_roberta_models.py (hunk 4 above)
Comment (jbischof) [reply keras_nlp/models/xlm_roberta/xlm_roberta_models.py:25]: This should just say `"""XLM-RoBERTa encoder."""`
